src/main.rs

```diff
@@ -31,6 +31,11 @@ fn print_emoji_selector<W: Write>(log_update: &mut LogUpdate<W>, selected: &Comm
     log_update.render(&text).unwrap();
 }
 
+fn get_emoji_at_index (index: u8) -> CommitType {
```

Comment:
I think that this would be nicer as an `at_index` extension to `CommitType`, and I think it should return an `Option<CommitType>` since there isn't a valid commit type for every valid `u8`